src/csharp/Grpc.Core/Internal/PlatformApis.cs

```diff
@@ -53,7 +53,7 @@ internal static class PlatformApis
 
         static PlatformApis()
         {
-#if NETSTANDARD1_5
+#if !NET45
```

Comment:
The tests now seem to be be able to find RuntimeInformation. Possibly building on a later .NET version?